h5p-bildetema/src/components/Button/Button.module.scss

```diff
@@ -22,7 +22,7 @@
 
 .secondary {
   color: black;
-  background-color: $white;
+  background-color: $beige;
```

Comment:
Keep $white perhaps 🤔

Reply:
This is because we changed dialog to white